src/ui/qgsprojectpropertiesbase.ui

```diff
@@ -2864,6 +2864,13 @@
                        </property>
                       </widget>
                      </item>
+                     <item row="9" column="0">
+                      <widget class="QCheckBox" name="mSkipNameForGroupCheckBox">
+                       <property name="text">
+                        <string>Skip name attribute for groups</string>
```

Comment:
Can you try to add a tooltip explaining the checkbox ? More or less inspired by the PR description ?

Because honestly, it can be difficult sometimes to understand some checkbox in this dialog, not your fault, just thinking about that just now ;-)

Reply:
No problem. Is the text "GetCapabilities response would skip name attribute for group items while leaving title attribute as is, thus showing the group but making it impossible to include the whole group (rather than it's elements) in subsequent GetMap request LAYERS parameter." clear enough and is it's length ok?

Reply:
It's tricky to explain, I know :)
But at least, it's better than nothing, when we don't know the PR

Reply:
Fixed